### PR TITLE
Update job tree parsing

### PR DESCRIPTION
**Pull request by orlandohohmeier**
Align `ChronosUtil.parseJobs` and the fetch jobs calls with the latest API adjustments:

* Fix fetchJobs activeRuns embed param
* Change the job id/namespace matching to the new id pattern
* Add missing embed schedules param

/cc @jfurrow

### CODE REVIEW

**Comment by mesosphere-web** [comment]
Reviewer: @MatApple

**Comment by pierlo-upitup** [comment]
Shouldn't we update our fixtures too so the tests dont fail?

**Comment by orlandohohmeier** [comment]
I've updated the respective [fixture](https://github.com/dcos/dcos-ui/blob/4a30829d28f9a1c92979eb542db95a6009819f72/tests/_fixtures/chronos/jobs.json), did I missed something?

**Comment by pierlo-upitup** [comment]
I swear I didn't see it!

File: src/js/utils/ChronosUtil.js [hunk 1]
```diff
@@ -35,9 +34,9 @@ function addJob(parent, item, jobsAlreadyAdded) {
 
   // Get the parent id (e.g. /group) by matching every thing but the item
   // name including the preceding slash "/" (e.g. /id).
```
Comment (pierlo-upitup) [inline src/js/utils/ChronosUtil.js]: Comment needs an update. #thisiswhyihatecomments

**Comment by pierlo-upitup** [comment]
Merge as you please! 💯

File: src/js/utils/ChronosUtil.js (hunk 1 above)
Comment (orlandohohmeier) [inline src/js/utils/ChronosUtil.js]: ✔️

**Pull request merged**